_posts/2023-11-23-announcing-dp-social-inbox.md

```diff
@@ -0,0 +1,101 @@
+
+## Failures of Existing Social Media
+
+Big social media services, like X/Twitter and Instagram, centralize all control over the data, content, and interactions on their platforms. They moderate messages on their platforms with little to no transparency nor accountability, leading to posts and entire accounts being censored with no explanation. Additionally, corporate social media platforms have an insatiable hunger for data. They surveil every aspect of our interactions - from what we post, what we "like", who we follow, to even minute details of where our attention lingers. This data is used to feed their algorithms in order to maximize the influence, and therefore profitability, of the ads they show to their users. 
+
+Their intrusive practices extend into the timeline itself, by manipulating what posts appear and inserting "suggested" content that we never ask to see. Major social media platforms regularly comply with government requests to hand over personal data without a court warrant. And we have witnessed how an entire social media network can be sold to a billionaire with no regard to the impact that would have on global public discourse. 
+
+## ActivityPub: A Decentralized Alternative
+
+In the face of the failures of existing social media platforms, there have been many attempts to build viable alternatives. Out of these, one of the most thriving networks is what is collectively called the Fediverse — an ecosystem of social media platforms that rely on the ActivityPub social data standard. 
+
+Instead of having a platform contain all the accounts, messages, and interactions all concentrated on one website, ActivityPub allows users with accounts on different websites to communicate with each other. Email works similarly in that you can send an email to someone else regardless of what email service they use. By enabling anyone to set up their own "instance" or social media server, ActivityPub allows people to decide who has control over all their data. They can decide to host their own instance, or choose to go with any of the many others who offer hosting services. The standard allows data to be hosted across thousands of different instances, while still being able to follow and send messages between them. 
+
+One of the perennial problems with social media is content moderation. What makes ActivityPub distinct is that it allows each instance to decide for themselves how to moderate messages on their servers. This helps ensure that a given user or community can shape their own moderation policies, instead of an unaccountable, centralized authority. Fediverse instances can even decide to subscribe to community-curated blocklists, which can be actively shaped and used across many servers, by people who understand their community needs and agree on what kind of dialogue they would like to have or not have appear in their timelines.
```

Comment:
```suggestion
One of the perennial problems with social media is content moderation. ActivityPub is distinct in that it allows each instance to decide for themselves how to moderate messages on their servers. This helps ensure that a given user or community can shape their own moderation policies, instead of an unaccountable, centralized authority. Fediverse instances can even decide to subscribe to community-curated blocklists, which can be actively shaped and used across many servers, by people who understand their community needs and agree on what kind of dialogue they would like to have or not have appear in their timelines.
```